llama_index/vector_stores/azureaisearch.py

```diff
@@ -141,68 +148,69 @@ def _create_index(self, index_name: Optional[str]) -> None:
             SearchField(
                 name=self._field_mapping["embedding"],
                 type=SearchFieldDataType.Collection(SearchFieldDataType.Single),
-                hidden=False,
                 searchable=True,
-                filterable=False,
-                sortable=False,
-                facetable=False,
                 vector_search_dimensions=self.embedding_dimensionality,
-                vector_search_configuration="default",
+                vector_search_profile_name="myHnswProfile",
```

Comment:
does the default profile name need to be like this

Reply:
no, it can be named anything

Reply:
could we just use `default`

Reply:
I'm OK with that